### PR TITLE
[CCI] Normalize interlink URL usage

### PR DESCRIPTION
**Pull request by SergeyMyssak**
### Description
Normalized interlink URL usage
 
### Issues Resolved
https://github.com/opensearch-project/oui/issues/698
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by SergeyMyssak** [comment]
Hello @KrooshalUX, I replaced all /1.0/ references with /latest/. I have also changed google.com and opensearch.org in some places, could you please check whether I should keep these changes or undo them?

File: src-docs/src/views/datagrid/in_memory.js [hunk 1]
```diff
@@ -50,12 +50,14 @@ for (let i = 1; i < 100; i++) {
       '{{name.lastName}}, {{name.firstName}} {{name.suffix}}'
     ),
     email: (
-      <OuiLink href="">{faker.helpers.fake('{{internet.email}}')}</OuiLink>
+      <OuiLink href="https://oui.opensearch.org/latest/">
+        {faker.helpers.fake('{{internet.email}}')}
+      </OuiLink>
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/in_memory.js:55]: Screenshot 2023-04-18 at 09 01 12

File: src-docs/src/views/datagrid/in_memory.js [hunk 2]
```diff
@@ -50,12 +50,14 @@ for (let i = 1; i < 100; i++) {
...
     ),
     location: (
       <Fragment>
         {`${faker.helpers.fake('{{address.city}}')}, `}
-        <OuiLink href="https://google.com">
+        <OuiLink href="https://oui.opensearch.org/latest/">
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/in_memory.js:60]: Screenshot 2023-04-18 at 09 01 27

File: src-docs/src/views/datagrid/in_memory_enhancements.js [hunk 3]
```diff
@@ -49,12 +49,14 @@ for (let i = 1; i < 100; i++) {
       '{{name.lastName}}, {{name.firstName}} {{name.suffix}}'
     ),
     email: (
-      <OuiLink href="">{faker.helpers.fake('{{internet.email}}')}</OuiLink>
+      <OuiLink href="https://oui.opensearch.org/latest/">
+        {faker.helpers.fake('{{internet.email}}')}
+      </OuiLink>
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/in_memory_enhancements.js:54]: Screenshot 2023-04-18 at 09 02 14

File: src-docs/src/views/datagrid/in_memory_enhancements.js [hunk 4]
```diff
@@ -49,12 +49,14 @@ for (let i = 1; i < 100; i++) {
...
     ),
     location: (
       <Fragment>
         {`${faker.helpers.fake('{{address.city}}')}, `}
-        <OuiLink href="https://google.com">
+        <OuiLink href="https://oui.opensearch.org/latest/">
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/in_memory_enhancements.js:59]: Screenshot 2023-04-18 at 09 02 33

File: src-docs/src/views/datagrid/in_memory_pagination.js [hunk 5]
```diff
@@ -49,12 +49,14 @@ for (let i = 1; i < 100; i++) {
       '{{name.lastName}}, {{name.firstName}} {{name.suffix}}'
     ),
     email: (
-      <OuiLink href="">{faker.helpers.fake('{{internet.email}}')}</OuiLink>
+      <OuiLink href="https://oui.opensearch.org/latest/">
+        {faker.helpers.fake('{{internet.email}}')}
+      </OuiLink>
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/in_memory_pagination.js:54]: Screenshot 2023-04-18 at 09 03 09

File: src-docs/src/views/datagrid/in_memory_pagination.js [hunk 6]
```diff
@@ -49,12 +49,14 @@ for (let i = 1; i < 100; i++) {
...
     ),
     location: (
       <Fragment>
         {`${faker.helpers.fake('{{address.city}}')}, `}
-        <OuiLink href="https://google.com">
+        <OuiLink href="https://oui.opensearch.org/latest/">
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/in_memory_pagination.js:59]: Screenshot 2023-04-18 at 09 03 18

File: src-docs/src/views/datagrid/in_memory_sorting.js [hunk 7]
```diff
@@ -49,12 +49,14 @@ for (let i = 1; i < 100; i++) {
       '{{name.lastName}}, {{name.firstName}} {{name.suffix}}'
     ),
     email: (
-      <OuiLink href="">{faker.helpers.fake('{{internet.email}}')}</OuiLink>
+      <OuiLink href="https://oui.opensearch.org/latest/">
+        {faker.helpers.fake('{{internet.email}}')}
+      </OuiLink>
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/in_memory_sorting.js:54]: Screenshot 2023-04-18 at 09 03 51

File: src-docs/src/views/datagrid/in_memory_sorting.js [hunk 8]
```diff
@@ -49,12 +49,14 @@ for (let i = 1; i < 100; i++) {
...
     ),
     location: (
       <Fragment>
         {`${faker.helpers.fake('{{address.city}}')}, `}
-        <OuiLink href="https://google.com">
+        <OuiLink href="https://oui.opensearch.org/latest/">
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/in_memory_sorting.js:59]: Screenshot 2023-04-18 at 09 04 04

File: src-docs/src/views/datagrid/virtualization.js [hunk 9]
```diff
@@ -82,11 +82,13 @@ function RenderCellValue({ rowIndex, columnId }) {
     const suffix = faker.helpers.fake('{{name.suffix}}');
     data[rowIndex] = {
       name: `${name} ${suffix}`,
-      email: <OuiLink href="">{email}</OuiLink>,
+      email: (
+        <OuiLink href="https://oui.opensearch.org/latest/">{email}</OuiLink>
+      ),
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/virtualization.js:87]: Screenshot 2023-04-18 at 09 04 53

File: src-docs/src/views/datagrid/virtualization.js [hunk 10]
```diff
@@ -82,11 +82,13 @@ function RenderCellValue({ rowIndex, columnId }) {
...
       location: (
         <Fragment>
           {`${faker.helpers.fake('{{address.city}}')}, `}
-          <OuiLink href="https://google.com">
+          <OuiLink href="https://oui.opensearch.org/latest/">
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/virtualization.js:91]: Screenshot 2023-04-18 at 09 05 18

File: src-docs/src/views/datagrid/virtualization_constrained.js [hunk 11]
```diff
@@ -80,11 +80,13 @@ function RenderCellValue({ rowIndex, columnId }) {
     const suffix = faker.helpers.fake('{{name.suffix}}');
     data[rowIndex] = {
       name: `${name} ${suffix}`,
-      email: <OuiLink href="">{email}</OuiLink>,
+      email: (
+        <OuiLink href="https://oui.opensearch.org/latest/">{email}</OuiLink>
+      ),
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/virtualization_constrained.js:85]: Screenshot 2023-04-18 at 09 05 50

File: src-docs/src/views/datagrid/virtualization_constrained.js [hunk 12]
```diff
@@ -80,11 +80,13 @@ function RenderCellValue({ rowIndex, columnId }) {
...
       location: (
         <Fragment>
           {`${faker.helpers.fake('{{address.city}}')}, `}
-          <OuiLink href="https://google.com">
+          <OuiLink href="https://oui.opensearch.org/latest/">
```
Comment (SergeyMyssak) [inline src-docs/src/views/datagrid/virtualization_constrained.js:89]: Screenshot 2023-04-18 at 09 06 10

File: src-docs/src/views/flyout/flyout_banner.js [hunk 13]
```diff
@@ -35,7 +35,8 @@ export default () => {
     <OuiCallOut iconType="help">
       <p>
         Here&rsquo;s some stuff that you need to know. This banner helps
-        highlight important information. <OuiLink href="#">View docs</OuiLink>
+        highlight important information.{' '}
+        <OuiLink href="https://oui.opensearch.org/latest/">View docs</OuiLink>
```
Comment (SergeyMyssak) [inline src-docs/src/views/flyout/flyout_banner.js:39]: Screenshot 2023-04-18 at 09 07 02

File: src-docs/src/views/header/header.js [hunk 14]
```diff
@@ -38,7 +38,7 @@ export default () => {
   const renderLogo = () => (
     <OuiHeaderLogo
       iconType="logoOpenSearch"
-      href="#"
+      href="https://oui.opensearch.org/latest/"
```
Comment (SergeyMyssak) [inline src-docs/src/views/header/header.js:41]: Hover over the logo

Screenshot 2023-04-18 at 09 07 36

File: src-docs/src/views/header/header_alert.js [hunk 15]
```diff
@@ -48,7 +48,11 @@ const HeaderUpdates = () => {
     {
       title: 'Control access to features',
       text: 'Show or hide applications and features per space in Kibana.',
-      action: <OuiLink href="">Learn about feature controls</OuiLink>,
+      action: (
+        <OuiLink href="https://oui.opensearch.org/latest/">
+          Learn about feature controls
+        </OuiLink>
+      ),
```
Comment (SergeyMyssak) [inline src-docs/src/views/header/header_alert.js:55]: Screenshot 2023-04-18 at 09 10 49

File: src-docs/src/views/header/header_alert.js [hunk 16]
```diff
@@ -71,7 +75,11 @@ const HeaderUpdates = () => {
       title: 'Enter dark mode',
       text:
         'Kibana now supports the easy-on-the-eyes theme across the entire UI.',
-      action: <OuiLink href="">Go to Advanced Settings</OuiLink>,
+      action: (
+        <OuiLink href="https://oui.opensearch.org/latest/">
+          Go to Advanced Settings
+        </OuiLink>
+      ),
```
Comment (SergeyMyssak) [inline src-docs/src/views/header/header_alert.js:82]: Screenshot 2023-04-18 at 09 11 28

File: src-docs/src/views/header/header_elastic_pattern.js [hunk 17]
```diff
@@ -283,7 +283,9 @@ export default ({ theme }) => {
         sections={[
           {
             items: [
-              <OuiHeaderLogo iconType="logoOpenSearch" href="">
+              <OuiHeaderLogo
+                iconType="logoOpenSearch"
+                href="https://oui.opensearch.org/latest/">
```
Comment (SergeyMyssak) [inline src-docs/src/views/header/header_elastic_pattern.js:288]: Hover over the logo

Screenshot 2023-04-18 at 09 15 48

File: src-docs/src/views/header/header_example.js [hunk 18]
```diff
@@ -95,16 +95,16 @@ const headerLinksSnippet = `<OuiHeader>
...
   </OuiHeaderSectionItem>
 
   <OuiHeaderLinks>
-    <OuiHeaderLink href="" isActive>
+    <OuiHeaderLink href="https://oui.opensearch.org/latest/" isActive>
       <!-- First link -->
     </OuiHeaderLink>
 
-    <OuiHeaderLink href="">
+    <OuiHeaderLink href="https://oui.opensearch.org/latest/">
```
Comment (SergeyMyssak) [inline src-docs/src/views/header/header_example.js:107]: Screenshot 2023-04-18 at 09 18 03

File: src-docs/src/views/header/header_sections.js [hunk 19]
```diff
@@ -28,7 +28,7 @@ export default () => {
   const renderLogo = (
     <OuiHeaderLogo
       iconType="logoOpenSearch"
-      href="#"
+      href="https://oui.opensearch.org/latest/"
```
Comment (SergeyMyssak) [inline src-docs/src/views/header/header_sections.js:31]: Hover over the logo

Screenshot 2023-04-18 at 09 19 09

File: src-docs/src/views/link/link.js [hunk 20]
```diff
@@ -17,15 +17,15 @@ export default () => (
...
+      <OuiLink href="https://oui.opensearch.org/latest/" target="_blank">
         OpenSearch website
       </OuiLink>{' '}
       in a new tab. Setting <OuiCode>target=&ldquo;_blank&rdquo;</OuiCode> also
       defaults to <OuiCode>{'external={true}'}</OuiCode>.
     </p>
     <p>
       This{' '}
-      <OuiLink href="https://Opensearch.org" external>
+      <OuiLink href="https://oui.opensearch.org/latest/" external>
```
Comment (SergeyMyssak) [inline src-docs/src/views/link/link.js:28]: Screenshot 2023-04-18 at 09 22 19

File: src-docs/src/views/link/link_example.js [hunk 21]
```diff
@@ -33,9 +33,9 @@ const linkValidationSource = require('!!raw-loader!./link_validation');
 const linkValidationHtml = renderToHtml(LinkValidation);
 
 const linkSnippet = [
-  `<OuiLink href="#"><!-- Link text --></OuiLink>
+  `<OuiLink href="https://oui.opensearch.org/latest/"><!-- Link text --></OuiLink>
 `,
-  `<OuiLink href="#" color="success">
+  `<OuiLink href="https://oui.opensearch.org/latest/" color="success">
```
Comment (SergeyMyssak) [inline src-docs/src/views/link/link_example.js:38]: Screenshot 2023-04-18 at 09 22 58

File: src-docs/src/views/suggest/hashtag_popover.js [hunk 22]
```diff
@@ -80,7 +80,7 @@ export default (props) => {
                 iconType: 'trash',
                 iconSize: 's',
               }}
-              href="#"
+              href="https://oui.opensearch.org/latest/"
```
Comment (SergeyMyssak) [inline src-docs/src/views/suggest/hashtag_popover.js:83]: Screenshot 2023-04-18 at 09 26 21

File: src-docs/src/views/tables/auto/auto.js [hunk 23]
```diff
@@ -114,7 +114,7 @@ const customColumns = [
     field: 'lastName',
     name: 'Last Name',
     render: (name) => (
-      <OuiLink href="#" target="_blank">
+      <OuiLink href="https://oui.opensearch.org/latest/" target="_blank">
```
Comment (SergeyMyssak) [inline src-docs/src/views/tables/auto/auto.js:117]: Screenshot 2023-04-18 at 09 27 22

File: src-docs/src/views/tables/basic/basic.js [hunk 24]
```diff
@@ -70,7 +70,7 @@ export const Table = () => {
       name: 'Last Name',
       truncateText: true,
       render: (name) => (
-        <OuiLink href="#" target="_blank">
+        <OuiLink href="https://oui.opensearch.org/latest/" target="_blank">
```
Comment (SergeyMyssak) [inline src-docs/src/views/tables/basic/basic.js:73]: Screenshot 2023-04-18 at 09 28 00

File: src-docs/src/views/tables/custom/custom.js [hunk 25]
```diff
@@ -587,15 +587,21 @@ export default class extends Component {
             ? item.title.value
             : item.title;
           const title = item.title.isLink ? (
-            <OuiLink href="">{item.title.value}</OuiLink>
+            <OuiLink href="https://oui.opensearch.org/latest/" target="_blank">
+              {item.title.value}
+            </OuiLink>
```
Comment (SergeyMyssak) [inline src-docs/src/views/tables/custom/custom.js:592]: Screenshot 2023-04-18 at 09 29 11

File: src-docs/src/views/text/playground.js [hunk 26]
```diff
@@ -23,8 +23,8 @@ export const textConfig = () => {
     type: PropTypes.ReactNode,
     value: `<h1>This is Heading One</h1>
     <p>
-      Ea mollit ullamco cillum ipsum adipisicing ea aute id. Cillum <a href="#">unfashionable</a> amet 
-      proident irure Lorem irure consequat veniam. Excepteur exercitation ex officia 
+      Ea mollit ullamco cillum ipsum adipisicing ea aute id. Cillum <a href="https://oui.opensearch.org/latest/">unfashionable</a> amet
+      proident irure Lorem irure consequat veniam. Excepteur exercitation ex officia
```
Comment (SergeyMyssak) [inline src-docs/src/views/text/playground.js:27]: Screenshot 2023-04-18 at 09 31 18

File: src-docs/src/views/text/text.js [hunk 27]
```diff
@@ -19,9 +19,10 @@ export default () => (
       <h1>This is Heading One</h1>
       <p>
         Veniam et quis in dolor aliqua dolor laboris nostrud nostrud Lorem.
-        <a href="#">unfashionable</a> Irure et et adipisicing eu mollit. Ullamco
-        laborum cillum ea id occaecat cupidatat ex dolor consequat ex mollit do.
-        Irure commodo incididunt reprehenderit deserunt cillum quis ad nostrud!{' '}
+        <a href="https://oui.opensearch.org/latest/">unfashionable</a> Irure et
+        et adipisicing eu mollit. Ullamco laborum cillum ea id occaecat
+        cupidatat ex dolor consequat ex mollit do. Irure commodo incididunt
+        reprehenderit deserunt cillum quis ad nostrud!{' '}
```
Comment (SergeyMyssak) [inline src-docs/src/views/text/text.js:25]: Screenshot 2023-04-18 at 09 30 09

File: src-docs/src/views/text/text_color.js [hunk 28]
```diff
@@ -68,7 +68,7 @@ export default () => (
         Sometimes you need to color entire blocks of text, no matter what is in
         them. You can always apply color directly (versus using the separated
         component) to make it easy. Links should still{' '}
-        <a href="#">properly color</a>.
+        <a href="https://oui.opensearch.org/latest/">properly color</a>.
```
Comment (SergeyMyssak) [inline src-docs/src/views/text/text_color.js:71]: Screenshot 2023-04-18 at 09 31 53

File: src-docs/src/views/toast/success.js [hunk 29]
```diff
@@ -33,7 +33,10 @@ export default () => (
 
     <p>
       And some other stuff on another line, just for kicks. And{' '}
-      <OuiLink href="#">here&rsquo;s a link</OuiLink>.
+      <OuiLink href="https://oui.opensearch.org/latest/">
+        here&rsquo;s a link
+      </OuiLink>
+      .
```
Comment (SergeyMyssak) [inline src-docs/src/views/toast/success.js:39]: Screenshot 2023-04-18 at 09 32 27

File: src-docs/src/views/toast/toast_list.js [hunk 30]
```diff
@@ -62,7 +62,10 @@ export default () => {
             </p>
             <p>
               And some other stuff on another line, just for kicks. And{' '}
-              <OuiLink href="#">here&rsquo;s a link</OuiLink>.
+              <OuiLink href="https://oui.opensearch.org/latest/">
+                here&rsquo;s a link
+              </OuiLink>
+              .
```
Comment (SergeyMyssak) [inline src-docs/src/views/toast/toast_list.js:68]: Screenshot 2023-04-18 at 09 33 05

File: src-docs/src/views/tool_tip/tool_tip.js [hunk 31]
```diff
@@ -27,7 +27,7 @@ export default () => (
       <p>
         This tooltip appears on the{' '}
         <OuiToolTip position="top" content="Here is some tooltip text">
-          <OuiLink href="#">top</OuiLink>
+          <OuiLink href="https://oui.opensearch.org/latest/">top</OuiLink>
```
Comment (SergeyMyssak) [inline src-docs/src/views/tool_tip/tool_tip.js:30]: Screenshot 2023-04-18 at 09 35 04

File: src-docs/src/views/accessibility/screen_reader.tsx [hunk 32]
```diff
@@ -49,7 +49,9 @@ export default () => (
       <p>
         This link is visible to all on focus:{' '}
         <OuiScreenReaderOnly showOnFocus>
-          <OuiLink href="#">Skip navigation</OuiLink>
+          <OuiLink href="https://oui.opensearch.org/latest/">
+            Skip navigation
+          </OuiLink>
```
Comment (SergeyMyssak) [inline src-docs/src/views/accessibility/screen_reader.tsx:54]: Screenshot 2023-04-18 at 09 42 21

File: src-docs/src/views/key_pad_menu/key_pad_beta.js [hunk 33]
```diff
@@ -19,21 +19,23 @@ import {
...
+      href="https://oui.opensearch.org/latest/"
       betaBadgeLabel="Experimental"
       betaBadgeTooltipContent="This module is not GA. Please help us by reporting any bugs.">
       <OuiIcon type="dashboardApp" size="l" />
     </OuiKeyPadMenuItem>
 
     <OuiKeyPadMenuItem
       label="Dashboard"
-      href="#"
+      href="https://oui.opensearch.org/latest/"
```
Comment (SergeyMyssak) [inline src-docs/src/views/key_pad_menu/key_pad_beta.js:38]: Screenshot 2023-04-18 at 09 47 19

File: src-docs/src/views/key_pad_menu/key_pad_menu.js [hunk 34]
```diff
@@ -19,19 +19,28 @@ import {
...
       <OuiIcon type="dashboardApp" size="l" />
     </OuiKeyPadMenuItem>
 
-    <OuiKeyPadMenuItem isDisabled label="Dashboard" href="#">
+    <OuiKeyPadMenuItem
+      isDisabled
+      label="Dashboard"
+      href="https://oui.opensearch.org/latest/">
       <OuiIcon type="dashboardApp" size="l" />
     </OuiKeyPadMenuItem>
```
Comment (SergeyMyssak) [inline src-docs/src/views/key_pad_menu/key_pad_menu.js:45]: Screenshot 2023-04-18 at 09 55 18

File: src-docs/src/views/key_pad_menu/key_pad_menu_example.js [hunk 35]
```diff
@@ -27,10 +27,10 @@ import KeyPadMenu from './key_pad_menu';
 const keyPadMenuSource = require('!!raw-loader!./key_pad_menu');
 const keyPadMenuHtml = renderToHtml(KeyPadMenu);
 const keyPadMenuSnippet = `<OuiKeyPadMenu>
-  <OuiKeyPadMenuItem label={label1} href="#">
+  <OuiKeyPadMenuItem label={label1} href="https://oui.opensearch.org/latest/">
     <OuiIcon type={icon1} size="l" />
   </OuiKeyPadMenuItem>
-  <OuiKeyPadMenuItem label={label2} href="#">
+  <OuiKeyPadMenuItem label={label2} href="https://oui.opensearch.org/latest/">
```
Comment (SergeyMyssak) [inline src-docs/src/views/key_pad_menu/key_pad_menu_example.js:33]: Screenshot 2023-04-18 at 09 51 01

File: src-docs/src/views/key_pad_menu/key_pad_menu_example.js [hunk 36]
```diff
@@ -51,7 +51,7 @@ const keyPadBetaSource = require('!!raw-loader!./key_pad_beta');
 const keyPadBetaHtml = renderToHtml(KeyPadBeta);
 const keyPadBetaSnippet = `<OuiKeyPadMenuItem
   label={label}
-  href="#"
+  href="https://oui.opensearch.org/latest/"
```
Comment (SergeyMyssak) [inline src-docs/src/views/key_pad_menu/key_pad_menu_example.js:54]: Screenshot 2023-04-18 at 09 51 23

File: src-docs/src/views/list_group/list_group_item_color.tsx [hunk 37]
```diff
@@ -29,15 +33,28 @@ export default () => (
...
 
     <OuiSpacer size="s" />
 
     <OuiListGroup style={{ background: 'black' }}>
-      <OuiListGroupItem href="#" label="Ghost" color="ghost" />
+      <OuiListGroupItem
+        href="https://oui.opensearch.org/latest/"
+        label="Ghost"
+        color="ghost"
+      />
```
Comment (SergeyMyssak) [inline src-docs/src/views/list_group/list_group_item_color.tsx:57]: Screenshot 2023-04-18 at 09 48 12

File: src-docs/src/views/badge/beta_badge.js [hunk 38]
```diff
@@ -64,13 +64,13 @@ export default () => (
     <OuiBetaBadge
       label="Visualize"
       iconType="visVisualBuilder"
-      href="https://playground.opensearch.org/app/visualize#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))"
+      href="https://oui.opensearch.org/latest/"
```
[the remark this answers is not in the document]
Comment (KrooshalUX) [reply src-docs/src/views/badge/beta_badge.js:67]: Please roll this one back, that link was used on purpose.

File: src-docs/src/views/badge/beta_badge.js [hunk 39]
```diff
@@ -64,13 +64,13 @@ export default () => (
...
       target="_blank"
     />
     &emsp;
     <OuiBetaBadge
       label="Experimental"
-      href="https://opensearch.org/docs/latest/"
+      href="https://oui.opensearch.org/latest/"
```
[the remark this answers is not in the document]
Comment (KrooshalUX) [reply src-docs/src/views/badge/beta_badge.js:73]: Please roll this one back, that link was used on purpose.

File: src-docs/src/views/button/button_as_link.js [hunk 40]
```diff
@@ -36,7 +36,7 @@ export default () => (
 
       <OuiFlexItem grow={false}>
         <OuiButtonIcon
-          href="https://opensearch.org/"
+          href="https://oui.opensearch.org/latest/"
```
[the remark this answers is not in the document]
Comment (KrooshalUX) [reply src-docs/src/views/button/button_as_link.js:39]: Please roll this one back, that link was used on purpose.

File: src-docs/src/views/call_out/danger.js [hunk 41]
```diff
@@ -17,7 +17,10 @@ export default () => (
   <OuiCallOut title="There was an error" color="danger" iconType="alert">
     <p>
       Now you have to fix it, but maybe{' '}
-      <OuiLink href="#">this link can help</OuiLink>.
+      <OuiLink href="https://oui.opensearch.org/latest/">
+        this link can help
+      </OuiLink>
+      .
```
[the remark this answers is not in the document]
Comment (KrooshalUX) [reply src-docs/src/views/call_out/danger.js:23]: The links in the call outs should go to https://opensearch.org/docs/latest/

File: src-docs/src/views/call_out/info.js [hunk 42]
```diff
@@ -25,7 +25,7 @@ export default () => (
       </p>
       <p>
         When possible, its recommended to include links to product{' '}
-        <OuiLink href="https://opensearch.org/docs/latest/">
+        <OuiLink href="https://oui.opensearch.org/latest/">
```
[the remark this answers is not in the document]
Comment (KrooshalUX) [reply src-docs/src/views/call_out/info.js:28]: Please roll this one back, that link was used on purpose.

File: src-docs/src/views/call_out/warning.js [hunk 43]
```diff
@@ -18,12 +18,10 @@ export default () => (
     <p>
       This feature is experimental, and should not be run in production. For
       more information on this feature, visit{' '}
-      <OuiLink href="https://opensearch.org/docs/latest/">
-        documentation
-      </OuiLink>
+      <OuiLink href="https://oui.opensearch.org/latest/">documentation</OuiLink>
```
[the remark this answers is not in the document]
Comment (KrooshalUX) [reply src-docs/src/views/call_out/warning.js:21]: Please roll this one back, that link was used on purpose.

File: src-docs/src/views/call_out/warning.js [hunk 44]
```diff
@@ -18,12 +18,10 @@ export default () => (
...
       .
     </p>
-    <OuiButton href="#" color="warning">
+    <OuiButton href="https://oui.opensearch.org/latest/" color="warning">
```
[the remark this answers is not in the document]
Comment (KrooshalUX) [reply src-docs/src/views/call_out/warning.js:24]: This link should be https://opensearch.org/docs/latest/ (we are demonstrating that in a call out to link to the opensearch documentation website)

File: src-docs/src/views/datagrid/additional_controls.js [hunk 45]
```diff
@@ -45,7 +45,7 @@ for (let i = 1; i < 20; i++) {
     ),
     email: faker.helpers.fake('{{internet.email}}'),
     city: (
-      <OuiLink href="http://google.com">
+      <OuiLink href="https://oui.opensearch.org/latest/">
```
[the remark this answers is not in the document]
Comment (KrooshalUX) [reply src-docs/src/views/datagrid/additional_controls.js:48]: Do the links to google here come from Faker? I am not sure if I remember.

File: src-docs/src/views/text/text.js (hunk 27 above)
Comment (KrooshalUX) [reply src-docs/src/views/text/text.js:25]: Did the . after "Lorem" get added by accident or was that already there? Apologies on increasing scope, but lets go ahead and link any other word in this paragraph and remove the "unfashionable" - I think this was leftover from some proactive quick changes to OUI Docs before release to get rid of old EUI/Elastic/Kibana content.

File: src-docs/src/views/toast/toast_list.js (hunk 30 above)
Comment (KrooshalUX) [reply src-docs/src/views/toast/toast_list.js:68]: Lets use https://opensearch.org/docs/latest/ for this demo.

File: src-docs/src/views/tool_tip/tool_tip.js (hunk 31 above)
Comment (KrooshalUX) [reply src-docs/src/views/tool_tip/tool_tip.js:30]: These should not be links, the # is on purpose to show a tooltip only.

File: src-docs/src/views/accessibility/screen_reader.tsx (hunk 32 above)
Comment (KrooshalUX) [reply src-docs/src/views/accessibility/screen_reader.tsx:54]: This skip navigation link is for accessibility, and should not have the URL associated with it.

**Comment by KrooshalUX** [comment]
@SergeyMyssak Thanks for the detailed look into this issue. I have commented on the changes where linking to OpenSearch Dashboard documentation was on purpose. There are a few other minor bits of feedback for the areas where previously a # was used in place of a URL as well.

File: src-docs/src/views/datagrid/additional_controls.js (hunk 45 above)
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply src-docs/src/views/datagrid/additional_controls.js:48]: They don't come from Faker. According to the history of commits, this change was made a long time ago - https://github.com/opensearch-project/oui/commit/b64d503e95837084417fc9005e663c7d8b145525

File: src-docs/src/views/accessibility/screen_reader.tsx (hunk 32 above)
Comment (SergeyMyssak) [reply src-docs/src/views/accessibility/screen_reader.tsx]: Reverted changes

Screenshot 2023-04-19 at 10 05 33

File: src-docs/src/views/badge/beta_badge.js (hunk 38 above)
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply src-docs/src/views/badge/beta_badge.js]: Reverted changes

Screenshot 2023-04-19 at 10 06 48

File: src-docs/src/views/badge/beta_badge.js (hunk 39 above)
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply src-docs/src/views/badge/beta_badge.js]: Reverted changes

Screenshot 2023-04-19 at 10 07 43

File: src-docs/src/views/button/button_as_link.js (hunk 40 above)
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply src-docs/src/views/button/button_as_link.js]: Reverted changes

Screenshot 2023-04-19 at 10 08 29

File: src-docs/src/views/call_out/danger.js (hunk 41 above)
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply src-docs/src/views/call_out/danger.js]: Made the changes according to the comment

Screenshot 2023-04-19 at 10 09 07

File: src-docs/src/views/text/text.js (hunk 27 above)
Comment (SergeyMyssak) [reply src-docs/src/views/text/text.js]: You are right, this word was taken from the previous text. Made the changes according to the comment

Screenshot 2023-04-19 at 10 10 39

File: src-docs/src/views/toast/toast_list.js (hunk 30 above)
Comment (SergeyMyssak) [reply src-docs/src/views/toast/toast_list.js]: Made the changes according to the comment

Screenshot 2023-04-19 at 10 12 03

File: src-docs/src/views/tool_tip/tool_tip.js (hunk 31 above)
Comment (SergeyMyssak) [reply src-docs/src/views/tool_tip/tool_tip.js]: Reverted changes

Screenshot 2023-04-19 at 10 12 35

File: src-docs/src/views/call_out/warning.js (hunk 43 above)
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply src-docs/src/views/call_out/warning.js]: Reverted changes

Screenshot 2023-04-19 at 10 35 54

File: src-docs/src/views/call_out/warning.js (hunk 44 above)
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply src-docs/src/views/call_out/warning.js]: Made the changes according to the comment

Screenshot 2023-04-19 at 10 36 20

File: src-docs/src/views/call_out/info.js (hunk 42 above)
[the remark this answers is not in the document]
Comment (SergeyMyssak) [reply src-docs/src/views/call_out/info.js]: Reverted changes

Screenshot 2023-04-19 at 12 22 43

**Comment by KrooshalUX** [comment]
@BSFishy I could be aligned with that- I think my main concern is just being able to demonstrate to non-engineering viewers of the documentation that links are supported. I am not sure # would communicate that on mouse over as well as an actual link - but  - for the sake of simplicity I am ok if we go that route.

**Comment by SergeyMyssak** [comment]
> @BSFishy I could be aligned with that- I think my main concern is just being able to demonstrate to non-engineering viewers of the documentation that links are supported. I am not sure # would communicate that on mouse over as well as an actual link - but - for the sake of simplicity I am ok if we go that route.

Hello @KrooshalUX, is there anything else about this PR that I should change or revert?

**Review by ashwin-pc — APPROVED** [review]
@SergeyMyssak This change is much appreciated!

@BSFishy Im okay with either the `#` or `oui.opensearch.org` path as long as we are consistent. I think i agree with Kroosh here to make it easier for non technical users to know what this value can hold but im not picky about it. Its a two way door decision anyways.

**Review by manasvinibs — APPROVED** [review]

**Review by BSFishy — APPROVED** [review]

**Pull request merged**